### PR TITLE
Removed legacy gov tx format

### PR DESCRIPTION
**Pull request by sampocs**
## Context
Removed the legacy gov tx handler and updated gov txs to the sdk 47 format.

**Note to reviewer**: I would recommend viewing this PR commit by commit

## Brief Changelog
* Updated protos to be normal msgs instead of gov proposals
* Added authority to keeper and moved gov txs to message server functions
* Updated simapp
* Fixed unit tests

### CODE REVIEW

File: ratelimit/keeper/keeper.go [hunk 1]
```diff
@@ -46,3 +49,8 @@ func NewKeeper(
 func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
+
+// GetAuthority returns the x/staking module's authority.
```
Comment (mpoke) [inline ratelimit/keeper/keeper.go]: nit: typo -- x/staking

File: ratelimit/keeper/keeper.go [hunk 2]
```diff
@@ -29,6 +30,7 @@ func NewKeeper(
 	cdc codec.BinaryCodec,
 	key storetypes.StoreKey,
 	ps paramtypes.Subspace,
+	authority string,
```
Comment (mpoke) [inline ratelimit/keeper/keeper.go:33]: Reflect this in the README (Integration section)

**Review by mpoke — APPROVED** [review]
LGTM

File: ratelimit/keeper/keeper.go (hunk 1 above)
Comment (sampocs) [reply ratelimit/keeper/keeper.go]: done

File: ratelimit/keeper/keeper.go (hunk 2 above)
Comment (sampocs) [reply ratelimit/keeper/keeper.go:33]: done

**Pull request merged**